config/dev.yaml

```diff
@@ -1,9 +1,12 @@
-ingress:
-    tls:
-      - secretName: "gloot-tls-cert"
-        hosts:
-          - vorlon-dev.gloot.com
-    hosts:
-      - host: vorlon-dev.gloot.com
-        paths:
-          - /
```

Comment:
you should remove the ingress part from `default.yaml` and you could add the common stuff from the routes to that file (everything but the match can live in `default.yaml`)